firefox-mobile/webpack.config.js

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// webpack config to bundle the Firefox for Android web extension from shared sources.
+
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+
+module.exports = {
+  entry: "./firefox-mobile/addon.js",
+  output: {
+    filename: "./dist/firefox-mobile/background.js"
+  },
+  plugins: [
+    new CopyWebpackPlugin([
+      {
+        from: "shared/content.js",
+        to: "./dist/firefox-mobile/content.js"
+      },
+      {
+        from: "shared/*.png",
+        to: "./dist/firefox-mobile/[name].[ext]"
+      },
+      {
+        from: "./firefox-mobile/manifest.json",
```

Comment:
I assume this should be `shared/manifest.json`, right? There is no manifest file in the `firefox-mobile` directory, so this fails.

Reply:
oh snap... that's actually a huge bug. there *is* a manifest.json there, but it's being ignored by gitignore -- so it's only local to my tree. >_<

Reply:
(and the reason i went for its own manifest.json is because it throws some errors at stuff it doesn't like, like default_icon, and doesn't need the contextMenu permission, so to avoid reviewers complaining at me... i'm serving a diff file)

Reply:
Ah! I didn't see any exceptions, but given I only tested on Fennec without looking at the console, no surprise. Everything is fine now! :)